modules/cms/classes/ComponentPartial.php

```diff
@@ -106,6 +109,19 @@ public static function loadOverrideCached($theme, $component, $fileName)
         if ($partial === null) {
             $partial = Partial::loadCached($theme, $component->alias . '/' . $fileName);
         }
+        
+        if ($partial === null && Config::get('cms.enableComponentThemeDefaultPartialFallback')) {
+            $classes = array_flip(ComponentManager::instance()->listComponents());
+            $default_alias = array_get($classes, Str::normalizeClassName($component->name));
```

Comment:
@Rike-cz you can just use `array_search` here instead. Also, as a convention, we use camelCase variables.
```suggestion
            $defaultAlias = array_search(ComponentManager::instance()->listComponents(), Str::normalizeClassName($component->name));
```